.github/ci.yaml

```diff
@@ -0,0 +1,33 @@
+name: build and test
+on:
+  push:
+    branches: [ "main" ]
```

Comment:
Does it mean that action won't trigger on any other branch except 'main'?

Reply:
yes